packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -132,4 +132,55 @@
     <_SDKImplicitReference Include="WindowsFormsIntegration"
                        Condition=" ('$(UseWindowsForms)' == 'true') And ('$(UseWPF)' == 'true') "/>
   </ItemGroup>
+  
+  
+  <!-- 
+        Supported (and unsupported) TargetFrameworks 
+        
+        Visual Studio Project System determines the list of valid TargetFrameworks to show 
+        in the Project properties by querying SupportedTargetFramework values. 
+        
+        The Project System does not refer to this list at this time for .NET Framework TFM's.
+  -->
+
+  <!-- 
+    When WindowsDesktop SDK is used without setting UseWPF or UseWinForms, it shows a (suppressible) warning and functions much
+    like Microsoft.NET.Sdk
+    
+    Likewise, when WindowsDesktop SDK is used with a netcore TFM that is less than 3.0, it will simply act as if it were an
+    Microsoft.NET.Sdk project (and show a suppressible build-time warning). 
+    
+    Detect these situations and skip updates to @(SupportedTargetFramework) etc. 
+  --> 
+  <ItemGroup Condition="('$(UseWPF)' == 'true' Or '$(UseWindowsForms)' == 'true') And 
+                        '$(_TargetFrameworkVersionWithoutV)' != '' And '$(_TargetFrameworkVersionWithoutV)' >= '$(_WindowsDesktopSdkTargetFrameworkVersionFloor)'">
+
+    <!-- 
+        Windows Forms and WPF are supported only on .NET Core 3.0+
+    -->
+    <_UnsupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v1.0" />
+    <_UnsupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v1.1" />
+    <_UnsupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v2.0" />
+    <_UnsupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v2.1" />
+    <_UnsupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v2.2" />
+
+    <!--
+        Windows Forms and WPF are not supported an any .NET Standard targets
+    -->
+    <_UnsupportedNETStandardTargetFramework Include="@(SupportedNETStandardTargetFramework)" />
+
+    <!--
+        Windows Forms and WPF are supported on .NET Framework 3.0+
```

Comment:
I would expect Winforms to work in 2.0.

Reply:
I missed that - fixing...

Reply:
oh, but just realized that we made another design decision before - that when using WindowsDesktop SDK, we don't support net20... 

that's how _WindowsDesktopSdkTargetFrameworkVersionFloor gets defined as 3.0 - as a common floor for netfx and netcore.

Reply:
I don't really care about 2.0, but if it were free to support it, it would be more accuratet.

Reply:
See https://github.com/dotnet/wpf/pull/1027#discussion_r296308910

Reply:
I improved the comment about what is supported, and why we are setting 3.0 as the lowest supported version in WindowsDesktop SDK. Still not adding 2.0 though.